website/blog/2019-08-31-dandelion-overview.md

```diff
@@ -1,73 +1,73 @@
 
-The fundamental mechanism that defines MolochDAO is the ability for members to exit the organization with a proportional share of the organization's assets, so long as they have not recently voted to approve a pending proposal. This creates a *loose partnership* where members can exit before a bad or otherwise controversial decision is made, and because of that guarantee there is less friction for collaboration in the first place.
+Why did we take this approach? MolochDAO was developed with a philosophy of minimizing complexity and code footprint. The organization does not try and do everything, but what it does do it does very well. In contrast, Aragon has been built with a philosophy of modularity and code-reuse. The Aragon ecosystem is growing a library of useful, audited, off-the-shelf apps that can be used in any Aragon DAO. After reviewing the [MolochDAO specification](https://github.com/MolochDAOVentures/MolochDAO) we determined the key features of MolochDAO that we wanted to replicate and set out to build them as Aragon apps. This would allow any Aragon organization to adopt similar mechanisms while also staying true to the Aragon development philosophy of creating modular and re-usable components.
 
-MolochDAO was developed with a philosophy of minimizing complexity and code footprint. The organization does not try and do everything, but what it does do it does very well. In contrast, Aragon has been built with a philosophy of modularity and code-reuse. As a result there is a growing library of useful, audited, off-the-shelf components that can be combined to create and expand a software-defined organization. After reviewing the [specification](https://github.com/MolochDAOVentures/MolochDAO) we determined that there were some key features of MolochDAO that we wanted to replicate in order to enable Aragon organization to adopt similar mechanisms, but stay true to the Aragon development philosophy of creating modular and re-usable components.
+The fundamental mechanism that defines MolochDAO is the ability for members to exit the organization with a proportional share of the organization's assets, so long as they have not recently voted to approve a pending proposal. This creates a *loose partnership* where members can exit before a bad or otherwise controversial decision is made. This guarantee reduces friction for collaboration in the first place.
 
-Specifically we wanted to make sure that we met the following requirements:
+In addition to providing this core feature, we are also enabling the following features:
```

Comment:
This changes the meaning significantly. This is not intended to be a features list, but actually are core requirements that are all critical to providing the guarantee.

Reply:
Changed